precompile/allow_list.go

```diff
@@ -23,9 +22,12 @@ const (
 )
 
 var (
-	AllowListNoRole  AllowListRole = AllowListRole(common.BigToHash(big.NewInt(0))) // No role assigned - this is equivalent to common.Hash{} and deletes the key from the DB when set
-	AllowListEnabled AllowListRole = AllowListRole(common.BigToHash(big.NewInt(1))) // Deployers are allowed to create new contracts
-	AllowListAdmin   AllowListRole = AllowListRole(common.BigToHash(big.NewInt(2))) // Admin - allowed to modify both the admin and deployer list as well as deploy contracts
+	// No role assigned - this is equivalent to common.Hash{} and deletes the key from the DB when set
+	AllowListNoRole AllowListRole = AllowListRole(common.BigToHash(common.Big0))
+	// Enabled - allowed to use state-changing precompile functions without modifying status of other admins or enableds
+	AllowListEnabled AllowListRole = AllowListRole(common.BigToHash(common.Big1))
+	// Admin - allowed to modify both the admin and enabled list, as well as to use state-changing precompile functions
+	AllowListAdmin AllowListRole = AllowListRole(common.BigToHash(common.Big2))
```

Comment:
is it necessary to write `AllowListAdmin AllowListRole = ...` or can the type be inferred eg `AllowListAdmin = ...`?

Reply:
yep good call